resources/deprecations.properties

```
@@ -4,6 +4,7 @@
 
 BlameSubversion = https://github.com/jenkinsci/jenkins/pull/5320
 blueocean-executor-info = https://issues.jenkins.io/browse/JENKINS-56773
+bmc-rpd-plugin = https://github.com/jenkins-infra/update-center2/pull/597
```

```suggestion
bmc-rpd = https://github.com/jenkins-infra/update-center2/pull/597
```

See URL: https://plugins.jenkins.io/bmc-rpd/